Lesson-3-(Strings-&-Arrays).md

```diff
@@ -1,4 +1,6 @@
-## In the prep work this week (chapters 7 & 8), the students learned:
+# Lesson 3: Strings and Arrays
+
+## In the prep work this week (Chapters 7 & 8), the students learned:
```

Comment:
maybe let's remove the chapter #s in case they change at any point?

Reply:
Done!